components/Explore.tsx

```diff
@@ -0,0 +1,110 @@
+import {router} from 'expo-router';
```

Comment:
Ditto - is this change in scope of this PR, or something @ItsMeKimi is going to submit as well?